Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -352,12 +355,19 @@ void _UIApplicationShutdown() {
 }
 
 extern "C" void UIApplicationMainHandleVoiceCommandEvent(IInspectable* voiceCommandResult) {
-    WMSSpeechRecognitionResult* result = [WMSSpeechRecognitionResult createWith:voiceCommandResult];
-    [[UIApplication sharedApplication] _sendVoiceCommandReceivedEvent:result];
+    id result = _createProjectionObject("WMSSpeechRecognitionResult", voiceCommandResult, "voice commands");
+
+    if (result) {
```

Comment:
Maybe so. If we do that, then we should also, for consistency, put an NSNull into the launchOptions dictionary.

---
In reply to: [112307083](https://github.com/Microsoft/WinObjC/pull/2569#discussion_r112307083) [](ancestors = 112307083)